tests/console/verify_systemd_timesync.pm

```diff
@@ -46,7 +46,7 @@ sub run {
     my $not_synced = 1;
     {
         do {
-            $not_synced = script_run("journalctl -u yast-timesync | grep \"Started One time sync configured by YaST\"");
+            $not_synced = script_run("journalctl -u yast-timesync | grep \"Finished One time sync configured by YaST\"");
```

Comment:
As per our discussion, would it make sense to match both words?
Other than that LGTM.

Reply:
Done, updated VR. I think it's unlikely that this test will run against the older version of systemd again, but , well, just-in-case cannot hurt.